instrumentation/apache-httpclient-4.0/build.gradle.kts

```diff
@@ -23,8 +23,9 @@ muzzle {
         // We want to support the dropwizard clients too.
         group = "io.dropwizard"
         module = "dropwizard-client"
-        versions = "(,)"
-        assertInverse = true
+        versions = "(,4.0.0)"
+        assertInverse = false
+        skipVersions.add("4.0.0")
```

Comment:
Do we need to skip if we are adding assertInverse to false?

Reply:
I thought we did but I just reran both httpclient and jaxrs and it passes without the skip.  I will remove.